internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -834,6 +834,20 @@ var (
 				},
 			},
 		},
+		{
+			Metadata: config.ChangeMetadata{
+				ID:          config.ChangeID("P135"),
+				Severity:    config.ChangeSeverityError,
+				Description: standardCoreEntityFieldsMessage("vault", []string{"*"}, versions300AndAbove),
```

Comment:
The [standardCoreEntityFieldsMessage](https://github.com/Kong/koko/blob/main/internal/server/kong/ws/config/compat/compatibility_table.go#L46-L55) looks more related to entities fields, but the message for Vaults should be about the whole entity. Something like (please rephrase in good inglish :D ):

```
func standardCoreEntityMessage(entityName string, versionWithFeatureSupport string) string {
	return fmt.Sprintf("The '%s' entity is being used, "+
		"but Kong Gateway versions < %s do not support this entity. "+
		entityName,
		versionWithFeatureSupport,
	)
}
```

WDYT?

Reply:
This is done